pello/pello_greeting.py

```diff
@@ -0,0 +1,5 @@
+def greeting():
+    print("Hello World!")
+
+if __name__ == "__main__":
+    greeting()
```

Comment:
This section, does it have any benefit? Do you call this file directly?

Reply:
From your task:
> obsahovat program "pello-greeting" 

I expected that there should be also executable file to run "pello_greeting". I'll remove it then.

Reply:
It should. Preferably via a console_script entypoint. Can get you some reading materials.